infrastructure/versions.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = ">= 5"
```

```suggestion
      version = "~> 5"
```

Or commit the lockfile. `>=` is a good default for reusable modules but root modules typically should be at least pinned to a given major.